plans/fix-dev-backend-oom-crash.md

```
@@ -0,0 +1,126 @@
+# fix(dev): backend OOMs during MulmoScript movie builds and takes the whole `yarn dev` down
+
+## Symptoms
+
+Mid-session the server vanishes. The client pane fills with:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)